package.json

```diff
@@ -38,6 +38,7 @@
     "apollo-link-http": "1.5.16",
     "bluebird": "3.7.2",
     "bootstrap": "3.4.1",
+    "canvas-confetti": "1.0.0",
```

Comment:
I was worried about this:

![image](https://user-images.githubusercontent.com/2205537/71425189-eee9ad00-2667-11ea-9b50-1567e1a03c46.png)

This is happening due to obfuscation in production builds. v1.0.1 was already released that fixes the obfuscation error.

Reply:
Oh fantastic, thanks @catdad!